eng/Sign.proj

```diff
@@ -41,6 +41,12 @@
       </FilesToSign>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(SignNugetPackages)' == 'true'">
+      <FilesToSign Include="$(OutDir)*.snupkg">
```

Comment:
You can collapse this into the above `ItemGroup`:

```xml
    <ItemGroup Condition="'$(SignNugetPackages)' == 'true'">
      <FilesToSign Include="$(OutDir)*.nupkg;$(OutDir)*.snupkg">
        <Authenticode>NuGet</Authenticode>
      </FilesToSign>
    </ItemGroup>
```

It saves a little bit on duplication.

Reply:
Thanks!